packages/file/index.js

```diff
@@ -24,8 +45,22 @@ function gzipSize (path) {
 
 let self = {
   name: '@size-limit/file',
-  async step60 (config, check) {
+  async step60 (config, check, nodeVersion = process.version) {
     let files = check.bundles || check.path
+
+    if (check.brotli === true) {
+      let isBrotliSupported = gte(nodeVersion, brotliRequiredNodeVersion)
+
+      if (isBrotliSupported) {
+        check.size = await sum(files, async i => brotliSize(i))
+
+        return
+      }
+      process.stderr.write(
+        chalk.yellow(`Brotli required node >= ${ brotliRequiredNodeVersion }\n`)
```

Comment:
In this case, it will be better to stop with an error, since we can’t show size with Brotli.

Reply:
fixed